util/start_test

```diff
@@ -27,6 +28,10 @@ from chplenv import *
 
 from test import activate_chpl_test_venv
 import argparse
+try:
+    import argcomplete
```

Comment:
Pretty slick, but why do we need this? For users with outdate virtual envs?

Reply:
Out of date virtualenv, or if they don't source chpl-scripts-completion. Hmm. actually so long as argcomplete is installed it shouldn't matter if chpl-scripts-completion was sourced (from a start_test working point of view that is, obviously tab completion wouldn't work.)

So yeah, just outdated virtualenv (and it's a core script, so I don't mind the extra checks for something that's not strictly needed for usability)